### DISCUSSION TITLE
Filter by reviewed/not-reviewed

### DISCUSSION THREAD

**Discussion by zinguru** [tolgee/tolgee-platform#1969] [Q&A] [answered]
Hello, I have a problem: 
let's say, I have over 1000 translated keys, which I need to review. 
Only way, to do this, is marking key as reviewed, scrolling through list of keys and watching color background differences, to determine which key has been already reviewed. 
I would be awesome  to be able filter keys by reviewed status, so I could concentrate on keys, that not been reviewed.

Thanks for you great free product and sorry for my bad english.

**Comment by cyyynthia** [comment, marked as answer]
Hey there, I believe what you're asking for is already possible!

![image](https://github.com/tolgee/tolgee-platform/assets/9999055/ffe3399f-09e0-4b25-b7e6-36778f490877)

**Reply by zinguru** [reply]
Oh, Thanks a lot!
